internal/connect/command_offset_describe.go

```diff
@@ -0,0 +1,138 @@
+package connect
+
+import (
+	"encoding/json"
+
+	"github.com/spf13/cobra"
+	"github.com/tidwall/pretty"
+
+	connectv1 "github.com/confluentinc/ccloud-sdk-go-v2/connect/v1"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+	"github.com/confluentinc/cli/v3/pkg/kafka"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+type OffsetConnectorOut struct {
```

Comment:
No need for this to be public
```suggestion
type humanOffsetConnectorOut struct {
```

Reply:
done